backend/tetra/data/models/result.py

```diff
@@ -272,16 +285,27 @@ def get_test_stats(
 
         results_table = cls.TABLE
         ands = [
+            cast(results_table.c.tags['false_positive'].astext, Boolean) == False,
```

Comment:
I would expect there's a `from sqlalchemy import cast, Boolean` missing at the top, but maybe somewhere there's some 🪄  importing it...